com.unity.formats.fbx/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "com.unity.formats.fbx",
   "displayName": "FBX Exporter",
-  "version": "4.1.0-pre.2",
+  "version": "4.1.0-pre.3",
   "dependencies": {
     "com.unity.timeline": "1.5.2",
-    "com.autodesk.fbx": "4.1.0-pre.1"
+    "com.autodesk.fbx": "4.1.0-pre.2"
```

Comment:
As mentioned in the other review, if we release com.autodesk.fbx let's bump the version to sync with the exporter package.